WordPress/Classes/ViewRelated/Media/MediaPicker/MediaPickerMenu+ImagePlayground.swift

```diff
@@ -0,0 +1,90 @@
+import UIKit
+import ImagePlayground
+
+extension MediaPickerMenu {
+    static var isImagePlaygroundAvailable: Bool {
+        guard #available(iOS 18.1, *) else {
+            return false
+        }
+        return ImagePlaygroundViewController.isAvailable
+    }
+
+    static var imagePlaygroundLocalizedTitle: String {
+        Strings.imagePlayground
+    }
+
+    func makeImagePlaygroundAction(delegate: ImagePlaygroundPickerDelegate) -> UIAction {
+        UIAction(
+            title: Strings.imagePlayground,
+            image: UIImage(systemName: "apple.image.playground"),
+            attributes: [],
+            handler: { _ in showImagePlayground(delegate: delegate) }
+        )
+    }
+
+    func showImagePlayground(delegate: ImagePlaygroundPickerDelegate) {
+        guard let presentingViewController else { return }
+
+        guard #available(iOS 18.1, *) else {
+            return wpAssertionFailure("Not available on this platform. Use `isImagePlaygroundAvailable`.")
+        }
+
+        let controller = _ImagePlaygroundController()
+        controller.delegate = delegate
+
+        let imagePlaygroundVC = ImagePlaygroundViewController()
+        imagePlaygroundVC.delegate = controller
+        objc_setAssociatedObject(imagePlaygroundVC, &MediaPickerMenu.strongDelegateKey, controller, .OBJC_ASSOCIATION_RETAIN_NONATOMIC)
```

Comment:
Just to double-check, should `controller` or `imagePlaygroundVC` be stored here?

Reply:
`controller` is the value that needs to be retained. `ImagePlaygroundViewController` is kept in memory by UIKit due to being presented on screen.

Reply:
So, this line should be ...?
```suggestion
        objc_setAssociatedObject(controller, &MediaPickerMenu.strongDelegateKey, controller, .OBJC_ASSOCIATION_RETAIN_NONATOMIC)
```

Reply:
The `value` to be retained is a third [parameter](https://developer.apple.com/documentation/objectivec/1418509-objc_setassociatedobject).

Reply:
🙈 Of course... Not sure how I misread that...

Reply:
I forget the signature literally every time I use it 😆